cms/envs/common.py

```diff
@@ -41,12 +41,8 @@
 from xmodule.modulestore.edit_info import EditInfoMixin
 
 ############################ FEATURE CONFIGURATION #############################
+STUDIO_NAME = "Studio"
```

Comment:
We seem to be standardizing on the term "Studio" within our codebase, so I think `STUDIO_NAME` is the best option. If at some point in the future we decide to standardize on "CMS" instead, that can be a separate PR that goes through the whole codebase, not just this variable.

As for i18n, I don't think that's a good idea, actually. The name of something like this is generally treated as a brand, and is kept untranslated.